### PR TITLE
Csmith tweaks

### PR DESCRIPTION
**Pull request by fitzgen**
r? @e00E or @emilio or @pepyakin whoever wants to get here first ;)

### CODE REVIEW

File: csmith-fuzzing/driver.py [hunk 1]
```diff
@@ -1,33 +1,94 @@
...
+            print("Error: '{}' exited with code {}".format(" ".join(cmd), result.returncode))
+            cat(stdout.name, title="stdout")
+            cat(stdout.name, title="stderr")
+        return result
+
+def run_bindgen(input, output):
+    return run_logged([
+        "bindgen",
+        "--with-derive-partialeq",
+        "--with-derive-eq",
```
Comment (pepyakin) [inline csmith-fuzzing/driver.py]: Maybe we should use all of the `--with-derive` to improve coverage?

Comment (fitzgen) [reply csmith-fuzzing/driver.py]: I'll file a follow up :)

**Comment by fitzgen** [comment]
Thanks for review! :)

**Comment by fitzgen** [comment]
Filed https://github.com/rust-lang-nursery/rust-bindgen/issues/1037

**Pull request merged**